src/SecondOrderSeawaterPolynomials.jl

```diff
@@ -35,6 +35,7 @@ seawater_polynomial(Θ, Sᴬ, Z) = ⋯ + R₁₁₀ * Sᴬ * Θ + ⋯
 ```
 """
 @Base.kwdef struct SecondOrderSeawaterPolynomial{FT} <: AbstractSeawaterPolynomial
+    R₀₀₀ :: FT = 0
```

Comment:
Is this needed?

Reply:
`R₀₀₀` is used in `SimplestRealisiticRoquetSeawaterPolynomial`. See https://github.com/CliMA/SeawaterPolynomials.jl/pull/28#issuecomment-1467070184 below.